treeherder/push_health/push_health.py

```diff
@@ -108,9 +119,19 @@ def get_push_health_test_failures(push, repository_ids):
     # find tests that have failed in the last 14 days
     # this is very cache-able for reuse on other pushes.
     option_map = OptionCollection.objects.get_option_collection_map()
-    prior_day = push.time.date() - datetime.timedelta(days=2)
-    intermittent_history = get_history(4, prior_day, intermittent_history_days, option_map, repository_ids)
-    fixed_by_commit_history = get_history(2, prior_day, fixed_by_commit_history_days, option_map, repository_ids)
+    push_date = push.time.date()
+    intermittent_history, cc = get_history(
```

Comment:
what's cc stand for? :)

Reply:
Ahh, sorry.  It's "cache_key".  So I should have made it ``ck``  or just ``cache_key``